code/index.html

```diff
@@ -1,22 +1,205 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<html id="top">
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+<head>
+  <meta charset="ut-8" />
+  <meta name="viewport" content="width=device-width">
+  <title>Space News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+</head>
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+<body>
+  <!--Header with nav-bar, background and animation-->
+  <header class="header">
+    <!--navbar with links and searchbar, collapses on mobile devices-->
+    <nav class="nav">
+      <input type="checkbox" id="checkbox">
+      <label for="checkbox"><a class="menu-button"></a>
+      </label>
+      <div class="menu-content">
+        <a href="contacts.html" target="blank">contact</a>
+        <a href="about.html" target="blank">about</a>
+        <a href="signup.html" target="blank">sign up</a>
+        <a href="login.html" target="blank">login</a>
+      </div>
+      <div class="search-content">
+        <input type="checkbox" id="searchbox">
+        <div class="searchbar">
+          <input type="text" placeholder="Search..">
+        </div>
+        <label for="searchbox">
+          <img class="search-button" src="./pictures/magnify.png">
+        </label>
+    </nav>
+    <!-- meteor animation -->
+    <div class="meteor"></div>
+    <!-- logo, normal and simpler version for mobile devices-->
+    <div class="logo"><img src="./pictures/logo_space.svg"></div>
+    <div class="logo-simple"><img src="./pictures/logo_space_simple.svg"></div>
+  </header>
+
+  <!--Top news, the most important news. Take up big space and top of the content-->
+  <section class="big-news">
+    <container class="big-img-container">
+      <img src="./pictures/astronaut.jpg">
+    </container>
+    <container class="big-text-container">
+      <h2>TOP NEWS</h2>
+      <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Id adipisci eum ipsa, facere atque ea a aut eligendi,
+        rerum placeat, quibusdam recusandae saepe reiciendis! Magni minus repudiandae eligendi sequi minima!
+        <br>
```

Comment:
We should avoid using `<br />` tags, you can read more about them in feedback document! (link attached in the main comment) - needs to be fixed